tc/sgx/trusted_worker_manager/enclave/wpe/signup_enclave_wpe.cpp

```diff
@@ -75,6 +75,41 @@ tcf_err_t ecall_GenerateNonce(uint8_t* out_nonce, size_t in_nonce_size) {
     return result;
 }  // ecall_GenerateNonce
 
+tcf_err_t ecall_VerifyUniqueIdSignature(const char* unique_id,
```

Comment:
unique_id -> unique_id_key. Just to emphasize that the unique id is a key. You could add a description what this function is doing and the parameters

Reply:
Done